### ISSUE TITLE
Blueprint fails with v2.0.0-beta.2

### ISSUE THREAD

**Issue by josemarluedke** [glimmerjs/glimmer.js#252]
It seems that `@glimmer/babel-plugin-glimmer-env` is not published.

```
ember new glimmer-v2 -b @glimmer/blueprint@2.0.0-beta.2
installing blueprint
  create .babelrc.js
  create .editorconfig
  create README.md
  create .gitignore
  create index.html
  create package.json
  create public/favicon.png
  create public/logo192.png
  create public/logo512.png
  create public/manifest.json
  create public/robots.txt
  create src/App.css
  create src/App.js
  create src/index.js
  create src/logo.svg
  create testem.json
  create tests/index.html
  create tests/rendering/App.test.js
  create tests/util.ts
  create tsconfig.json
  create webpack.config.js

🚧  Installing packages... This might take a couple of minutes.
Error creating new application. Removing generated directory `./glimmer-v2`
Command failed: yarn install --non-interactive
error Couldn't find any versions for "@glimmer/babel-plugin-glimmer-env" that matches "^1.0.0"

yarn install v1.19.1
info No lockfile found.
[1/5] Validating package.json...
[2/5] Resolving packages...
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by pzuraq** [comment]
Ah, it _is_ published, but it's published to v2.0.0-beta.2 😅 looks like we need to make sure our blueprint is updating automatically as well..

**Comment by josemarluedke** [comment]
This is now fixed with #261 and released in beta.3

Thanks.
